build-parent/pom.xml

```diff
@@ -374,7 +374,7 @@
             <dependency>
                 <groupId>me.escoffier.certs</groupId>
```

Comment:
Can this project be hosted at quarkus.io, or smallrye ?

Reply:
Not for now. We will move it when I'm done with what I have in mind.
However, the benefits are so nice (seriously, not fearing expiration, making sure that it's done right, removing trust all..., I feel so much more comfortable) that I started adopting it in various places.

Reply:
@cescoffier Sounds good, thanks